<ui version="4.0" >
 <class>profileEditPage</class>
 <widget class="QWidget" name="profileEditPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>452</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_6" >
     <property name="title" >
      <string>Profile Management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <layout class="QVBoxLayout" name="listLayout" >
          <item>
           <widget class="KListWidget" name="profilesList" >
            <property name="maximumSize" >
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QToolBox" name="toolBox" >
            <property name="currentIndex" >
             <number>0</number>
            </property>
            <widget class="QWidget" name="page" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>458</width>
               <height>292</height>
              </rect>
             </property>
             <attribute name="label" >
              <string>Actions</string>
             </attribute>
             <attribute name="toolTip" >
              <string>Configure actions</string>
             </attribute>
             <layout class="QFormLayout" name="formLayout" >
              <property name="fieldGrowthPolicy" >
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <item row="1" column="0" >
               <widget class="QCheckBox" name="disableCompositing" >
                <property name="text" >
                 <string>Disable desktop effects, if active</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QCheckBox" name="dimDisplayOnIdle" >
                <property name="toolTip" >
                 <string>Activate automatic dimming</string>
                </property>
                <property name="statusTip" >
                 <string/>
                </property>
                <property name="whatsThis" >
                 <string>Automatically dims the display when the system is idle.</string>
                </property>
                <property name="text" >
                 <string>Dim display when idle for more than</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" >
               <widget class="KIntSpinBox" name="dimOnIdleTime" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="suffix" >
                 <string comment="Minutes" > min</string>
                </property>
                <property name="minimum" >
                 <number>1</number>
                </property>
                <property name="maximum" >
                 <number>280</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0" >
               <widget class="QLabel" name="PoweredIdleLabel_2" >
                <property name="toolTip" >
                 <string/>
                </property>
                <property name="whatsThis" >
                 <string>To prevent data loss or other damage, you can have the system suspend or hibernate, so you do not run accidentally out of battery power. Configure the number of minutes below which the machine will run the configured action.</string>
                </property>
                <property name="text" >
                 <string>When the system is idle for more than</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="1" >
               <layout class="QHBoxLayout" name="horizontalLayout_4" >
                <item>
                 <widget class="KIntSpinBox" name="idleTime" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="suffix" >
                   <string> min</string>
                  </property>
                  <property name="minimum" >
                   <number>1</number>
                  </property>
                  <property name="maximum" >
                   <number>280</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="KComboBox" name="idleCombo" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize" >
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="whatsThis" >
                   <string>To prevent data loss or other damage, you can have the system suspend or hibernate, so you do not run accidentally out of battery power. Configure the number of minutes below which the machine will run the configured action.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0" >
               <widget class="QLabel" name="label_7" >
                <property name="toolTip" >
                 <string/>
                </property>
                <property name="text" >
                 <string>When laptop lid closed</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" >
               <widget class="KComboBox" name="laptopClosedCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="whatsThis" >
                 <string>This action will be performed when the laptop lid gets closed</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" >
               <widget class="QLabel" name="label" >
                <property name="text" >
                 <string>When power button pressed</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" >
               <widget class="KComboBox" name="powerButtonCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="6" column="0" >
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>When sleep button pressed</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1" >
               <widget class="KComboBox" name="sleepButtonCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>dimDisplayOnIdle</zorder>
             <zorder>dimOnIdleTime</zorder>
             <zorder>PoweredIdleLabel_2</zorder>
             <zorder>label_7</zorder>
             <zorder>laptopClosedCombo</zorder>
             <zorder>label</zorder>
             <zorder>powerButtonCombo</zorder>
             <zorder>label_2</zorder>
             <zorder>sleepButtonCombo</zorder>
             <zorder></zorder>
             <zorder>disableCompositing</zorder>
            </widget>
            <widget class="QWidget" name="page_3" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>408</width>
               <height>292</height>
              </rect>
             </property>
             <attribute name="label" >
              <string>Screen</string>
             </attribute>
             <attribute name="toolTip" >
              <string>Configure preferences for Screen Powersaving</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4" >
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout" >
                <item>
                 <widget class="QLabel" name="PoweredBrightnessLabel_2" >
                  <property name="toolTip" >
                   <string>With this slider you can set the brightness when the system is plugged into the socket outlet</string>
                  </property>
                  <property name="text" >
                   <string>Brightness:</string>
                  </property>
                  <property name="wordWrap" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="brightnessSlider" >
                  <property name="mouseTracking" >
                   <bool>true</bool>
                  </property>
                  <property name="acceptDrops" >
                   <bool>true</bool>
                  </property>
                  <property name="whatsThis" >
                   <string/>
                  </property>
                  <property name="maximum" >
                   <number>100</number>
                  </property>
                  <property name="singleStep" >
                   <number>1</number>
                  </property>
                  <property name="pageStep" >
                   <number>1</number>
                  </property>
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickInterval" >
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" >
                <item>
                 <widget class="QCheckBox" name="DPMSEnable" >
                  <property name="text" >
                   <string>&amp;Enable display power management</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3" >
                  <property name="orientation" >
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0" >
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="KUrlLabel" name="DPMSLabel" />
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2" >
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType" >
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout_3" >
                <item row="1" column="0" >
                 <widget class="QCheckBox" name="DPMSStandbyEnabled" >
                  <property name="whatsThis" >
                   <string>Choose the period of inactivity after which the display should enter "standby" mode. This is the first level of power saving.</string>
                  </property>
                  <property name="text" >
                   <string>&amp;Standby after</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="KIntSpinBox" name="DPMSStandbyTime" >
                  <property name="maximumSize" >
                   <size>
                    <width>150</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="whatsThis" >
                   <string>Choose the period of inactivity after which the display should enter "standby" mode. This is the first level of power saving.</string>
                  </property>
                  <property name="suffix" >
                   <string> min</string>
                  </property>
                  <property name="prefix" >
                   <string/>
                  </property>
                  <property name="maximum" >
                   <number>360</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QCheckBox" name="DPMSSuspendEnabled" >
                  <property name="text" >
                   <string>S&amp;uspend after</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="KIntSpinBox" name="DPMSSuspendTime" >
                  <property name="maximumSize" >
                   <size>
                    <width>150</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="suffix" >
                   <string> min</string>
                  </property>
                  <property name="maximum" >
                   <number>360</number>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" >
                 <widget class="QCheckBox" name="DPMSPowerOffEnabled" >
                  <property name="text" >
                   <string>&amp;Power off after</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1" >
                 <widget class="KIntSpinBox" name="DPMSPowerOffTime" >
                  <property name="maximumSize" >
                   <size>
                    <width>150</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="suffix" >
                   <string> min</string>
                  </property>
                  <property name="maximum" >
                   <number>360</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer" >
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2" >
             <property name="geometry" >
              <rect>
               <x>0</x>
               <y>0</y>
               <width>412</width>
               <height>292</height>
              </rect>
             </property>
             <attribute name="label" >
              <string>CPU and System</string>
             </attribute>
             <attribute name="toolTip" >
              <string>Configure System and CPU preferences</string>
             </attribute>
             <layout class="QFormLayout" name="formLayout_2" >
              <property name="fieldGrowthPolicy" >
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_12" >
                <property name="text" >
                 <string>Turn off the following CPU(s)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <layout class="QHBoxLayout" name="CPUListLayout" />
              </item>
              <item row="3" column="0" >
               <widget class="QLabel" name="PoweredFreqLabel_2" >
                <property name="toolTip" >
                 <string/>
                </property>
                <property name="whatsThis" >
                 <string>CPU Behaviour strongly impacts on your system performance and on your battery's life. Here you can choose the policy best suitable for this profile</string>
                </property>
                <property name="text" >
                 <string>CPU frequency scaling policy:</string>
                </property>
                <property name="wordWrap" >
                 <bool>false</bool>
                </property>
                <property name="buddy" >
                 <cstring>freqCombo</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="1" >
               <widget class="KComboBox" name="freqCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>250</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="whatsThis" >
                 <string>CPU Behaviour strongly impacts on your system performance and on your battery's life. Here you can choose the policy best suitable for this profile</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" >
               <widget class="QLabel" name="label_19" >
                <property name="text" >
                 <string>System powersaving scheme:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" >
               <widget class="KComboBox" name="schemeCombo" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>250</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="5" column="0" >
               <widget class="QLabel" name="label_20" >
                <property name="text" >
                 <string>When loading profile execute:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" >
               <widget class="KUrlRequester" name="scriptRequester" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>250</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="1" >
               <widget class="Line" name="line_5" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionNewProfile" >
   <property name="text" >
    <string>New Profile</string>
   </property>
  </action>
  <action name="actionDeleteProfile" >
   <property name="text" >
    <string>Delete Profile</string>
   </property>
  </action>
  <action name="actionImportProfiles" >
   <property name="text" >
    <string>Import Profiles</string>
   </property>
  </action>
  <action name="actionExportProfiles" >
   <property name="text" >
    <string>Export Profiles</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
